src/_locales/en_US/messages.json

```diff
@@ -121,6 +121,12 @@
   "viewDeviceLimit": {
     "message": "Turn off Firefox Private Network on another browser to use it here. It can be active on up to three browsers."
   },
+  "viewDeviceLimit2": {
+    "message": " (It may take up for the changes to take effect)"
```

Comment:
Is this missing some words? I assume a time goes in there like "It may take up to 15 minutes...." or something?